File: 0022/0022.cc
#include <iostream>
#include <time.h>

int main()
{
	/* This thing disables buffering, permitting the writing of individual
	   characters to the standard output. */
	std::cout.setf(std::ios::unitbuf);

	/* For all characters in the string... */
	for(char j : "We are the music makers.\n")
	{
	  /* The current character is written to the standard output. */
		std::cout << j;
		/* The program sleeps for 0.1 seconds before repeating the loop. */
		nanosleep((const struct timespec []) {0,100000000}, NULL);
	}
	main();
}

File: 0022/0022.pl
#!/usr/bin/perl

use Time::HiRes;

# Buffering is disabled, ensuring that characters are "immediately"
# written to the standard output.
$|=1;

# This loop is run forever... unless something goes horribly, horribly
# wrong.
while(true)
{
	# For all characters in the message which should be written...
	foreach $i (split //, "We are the music makers.\n")
	{
		# A character is written to the standard output.
		print($i);
		# The program sleeps for 0.1 seconds.
		Time::HiRes::usleep(100000);
	}
}
